### DISCUSSION TITLE
How to add a new column to telemetry_driver_1 dataframe?

### DISCUSSION THREAD

**Discussion by RapidDestroyer1** [theOehrly/Fast-F1#228] [Q&A] [answered]
I am wondering how to do so as I am trying to fix the problem with the ranging DRS values. 
I assume it is a dataframe?
Thanks.

**Comment by theOehrly** [comment, marked as answer]
Yes, it's just a subclass of a pandas DataFrame. So you can use all the methods that pandas provides on it.

**Reply by RapidDestroyer1** [reply]
Thanks @theOehrly 
How would it be best to do that? 
Also, how would I change an axis as at the moment I have the y axis in microseconds but I instead want it in seconds?

**Reply by RapidDestroyer1** [reply]
Think I'm good actually
